lib/Classifiers/Classifier.php

```diff
@@ -84,7 +84,7 @@ public function classifyFiles(string $model, array $queueFiles, int $timeout): \
 				$path = $this->getConvertedFilePath($files[0]);
 				if (in_array($model, [ImagenetClassifier::MODEL_NAME, LandmarksClassifier::MODEL_NAME, ClusteringFaceClassifier::MODEL_NAME], true)) {
 					// Check file data size
-					$filesizeMb = filesize($path) / (1024 * 1024);
+					$filesizeMb = filesize($path) / (self::TEMP_FILE_DIMENSION * self::TEMP_FILE_DIMENSION);
```

Comment:
This is about converting the value from bytes to MiB, though, not about pixel dimensions.

Reply:
Yes, thats on my back burner for now, trying to figure out the GetPreview thing.  There is a PHP setting for this, i was thinking about reading that one?